ajax/libs/jquery.turbolinks/package.json

```diff
@@ -1,8 +1,29 @@
 {
   "name": "jquery.turbolinks",
-  "version": "2.0.2",
-  "filename": "jquery.turbolinks.min.js",
-  "author": "Sasha Koss <kossnocorp@gmail.com>",
+  "version": "2.1.0",
+  "filename": "vendor/assets/javascripts/jquery.turbolinks.min.js",
   "description": "jQuery plugin for drop-in fix binded events problem caused by Turbolinks",
-  "license": "MIT"
+  "license": "MIT",
+  "keywords": [
+    "jquery",
+    "turbolinks",
+    "website"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/kossnocorp/jquery.turbolinks.git"
+  },
+  "npmName": "jquery.turbolinks",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "vendor/assets/javascripts/*"
```

Comment:
I think you can use "vendor/assets/javascripts/" as basepath and "jquery.turbolinks*" as files in npmFileMap, but I'm not very sure. @PeterDaveHello Would you please explain again that whether only "dist" or "build" can be basepath? Thank you.